tests/run_ipolates_tests.sh

```diff
@@ -6,10 +6,10 @@
 set -e
 
 echo "create a new grib file with integer values"
-../wgrib2/wgrib2 data/gdas.t12z.pgrb2.1p00.anl.75r.grib2 -match "(UGRD|VGRD)" -rpn floor -set_scaling 0 0 -grib_out test.grb
+../wgrib2/wgrib2 data/gdas.t12z.pgrb2.1p00.anl.75r.grib2 -match "(UGRD|VGRD)" -rpn floor -set_scaling 0 0 -grib_out new_grid_test.grb
 
 echo "*** Testing conversion from earth to grid"
-../wgrib2/wgrib2 test.grb -new_grid_winds grid  \
+../wgrib2/wgrib2 new_grid_test.grb -new_grid_winds grid  \
```

Comment:
Use the name of the test script in the name of any test files produced, like this:

run_ipolates_tests_new_grid_test.grb

Put the test script name first on all of them. Then, when looking at an ls of the test directory, all the test data produced by one test will be easy to see.